spec/all_pages_spec.rb

```diff
@@ -37,7 +37,7 @@ def get_locale(path)
         expect(doc).to link_to_valid_headers
       end
 
-      xit 'links to valid internal pages' do
+      it 'links to valid internal pages' do
```

Comment:
1. I *think* that htmlproofer might do this for us....?
2. This will implicitly check that `__LOCALE_BASE_URL__` isn't left in rendered pages accidentally, correct? One thought was we'd need an explicit check for that but I think it's covered by making sure the links are valid

Reply:
Yeah, it looks like htmlproofer does this for us.

Test:

```diff
diff --git a/content/_en/landing/create_an_account.md b/content/_en/landing/create_an_account.md
index 74dac5b..f968c1c 100644
--- a/content/_en/landing/create_an_account.md
+++ b/content/_en/landing/create_an_account.md
@@ -25,7 +25,7 @@ image: /assets/img/login-gov-600x314.png
           <div class="mobile step-2-img"></div>
         </div>
         <div class="step-3 step list">
-          {{ site.data[page.lang].settings["create-an-account"]["step_3"] | replace_locale_base_url | markdownify }}
+          {{ site.data[page.lang].settings["create-an-account"]["step_3"] | markdownify }}
           <div class="mobile step-3-img"></div>
         </div>
       </article>
```

```
$ make htmlproofer_internal
bundle exec htmlproofer --disable-external --allow-hash-href --empty_alt_ignore `pwd`/_site
Running ["ImageCheck", "LinkCheck", "ScriptCheck"] on ["/Users/andrewmduthie/Documents/Code/identity-site/_site"] on *.html... 


Ran on 546 files!


- /Users/andrewmduthie/Documents/Code/identity-site/_site/create-an-account/index.html
  *  internally linking to __LOCALE_BASE_URL__/help/authentication-methods/which-authentication-method-should-i-use/, which does not exist (line 443)
     <a href="__LOCALE_BASE_URL__/help/authentication-methods/which-authentication-method-should-i-use/">authentication methods</a>

HTML-Proofer found 1 failure!
make: *** [htmlproofer_internal] Error 1
```

Reply:
Just to keep things neat, I moved this to #658. I'll revert the changes here and deal with merge conflicts when they come.